docs/user-guide/doc-odm-user-guide/exporting-data.md

```diff
@@ -1,19 +1,33 @@
 # Exporting Data
 
-To export data from a study either click the three buttons icon next to the study title from the Study Browser or click the study name in the Metadata editor and select **Export data**.
+In addition to browsing, exploring, and visualizing studies, you can also export full datasets. 
 
-![image](doc-odm-user-guide/images/export-data-menu.png)
+1. In your study window click on the **Export Button**
+* The **Export** button can be accessed from the right side of the dashboard or by clicking on the study title, which will reveal a dropdown menu with the option to export data.
```

Comment:
We've added yet another one export button, so your screenshot is outdated comparing with the current develop branch (=1.59 release), could you please check fresh develop branch and update instruction and screenshot?